tests/test_jobs.py

```diff
@@ -0,0 +1,64 @@
+from .utils import MockResponse, BaseTest
+from pyintelowl.main import cli
+from unittest import mock
+from requests import Session
+
+
+class TestJobs(BaseTest):
+    def test_job_help(self):
+        result = self.runner.invoke(cli, ["jobs", "-h"])
+        assert "Manage Jobs" in result.output
```

Comment:
applies to all: should use the assert functions provided by the `unittest` framework.